package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "auth0-account-link-extension",
-  "version": "2.7.0",
+  "version": "2.7.1",
```

Comment:
I don't think this version bump is necessary since you are not changing anything in the extension itself. Also, the bundle.js probably doesn't need to be updated.